client/state/data-layer/wpcom/followers/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Internal dependencies
+ */
+import { registerHandlers } from 'calypso/state/data-layer/handler-registry';
+import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
+import { http } from 'calypso/state/data-layer/wpcom-http/actions';
+import { FOLLOWERS_REQUEST, FOLLOWER_REMOVE_REQUEST } from 'calypso/state/action-types';
+import {
+	requestFollowersSuccess,
+	requestFollowersFailure,
+	successRemoveFollower,
+	failRemoveFollower,
+} from 'calypso/state/followers/actions';
+
+const fetchFollowers = ( action ) => {
+	const { query } = action;
+
+	return [
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ query.siteId }/followers`,
+				apiVersion: '1.1',
+				query,
+			},
+			action
+		),
+	];
+};
+
+const receiveFollowers = ( { query }, data ) => [ requestFollowersSuccess( query, data ) ];
+
+const requestFollowersError = ( { query }, error ) => [ requestFollowersFailure( query, error ) ];
+
+const removeFollower = ( action ) => {
+	const { siteId, follower } = action;
+
+	return [
+		http(
+			{
+				method: 'POST',
+				path: `sites/${ siteId }/followers/${ follower.ID }/delete`,
```

Comment:
Removing followers currently won't work because this `path` is missing a leading `/`.